src/network/netdev/tuntap.c

```diff
@@ -234,6 +236,8 @@ static int tuntap_verify(NetDev *netdev, const char *filename) {
                 r = get_user_creds(&user, &uid, NULL, NULL, NULL, USER_CREDS_ALLOW_MISSING);
                 if (r < 0)
                         log_netdev_warning_errno(netdev, r, "Cannot resolve user name %s, ignoring: %m", t->user_name);
+                else if (!uid_is_system(uid))
+                        log_netdev_warning(netdev, "User '%s' is not a system user (UID="UID_FMT"), ignoring.", t->user_name, uid);
```

Comment:
this looks fine, but I figure sooner or later we should probably switch more things over to doing userdb_by_name() and then using user_record_disposition() to determine if the user is a system user. This has the benefit that user database provides can tightly control on their own how they want a user categorized.

Reply:
OK. I will try.